conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosIndexer.scala

```diff
@@ -183,6 +193,30 @@ class TezosIndexer private (
       } else emptyOutcome
   }
 
+  private def processLastForks(
+      maxIndexedLevel: BlockLevel,
+      depth: Long,
+      interval: Long,
+      iteration: Long
+  ): Future[Option[AccountResetEvents]] = {
+    lazy val emptyOutcome = Future.successful(Option.empty)
+    if (featureFlags.forkHandlingIsOn && maxIndexedLevel != indexedData.defaultBlockLevel && iteration % interval == 0)
+      backtrackingForkProcessor.handleForkFrom(maxIndexedLevel, depth).flatMap {
+        case None =>
+          println(s"AFH: No fork detected up to $maxIndexedLevel")
+          emptyOutcome
+        case Some((forkId, invalidations)) =>
+          println(
+            s"AFH: A fork was detected somewhere before the currently indexed level $maxIndexedLevel. $invalidations entries were invalidated and connected to fork $forkId"
```

Comment:
"local fork"

Reply:
Can we also print out the affected levels and block hashes for more logging info?

Reply:
changed, also changed to use logger